lib/src/frontend/spawn_hybrid.dart

```diff
@@ -20,7 +20,7 @@ import '../utils.dart';
 /// The spawned isolate sends three kinds of messages. Data messages are emitted
 /// as data events, error messages are emitted as error events, and print
 /// messages are printed using `print()`.
-final _transformer = new StreamChannelTransformer<Object, Map>(
+final _transformer = new StreamChannelTransformer<dynamic, dynamic>(
```

Comment:
maybe add a TODO or file an issue to see if we can restore the type here and flow the types through the rest of the code.

Reply:
Done.